sky/unit/test/widget/stack_test.dart

```diff
@@ -127,8 +134,8 @@ void main() {
 
       Widget buildFrame(int index) {
         itemsPainted = [];
-        List<Widget> items = new List.generate(itemCount, (i) {
-          return new CustomPaint(child: new Text('$i'), callback: (_0, _1) { itemsPainted.add(i); });
+        List<Widget> items = new List<Widget>.generate(itemCount, (i) {
+          return new CustomPaint(child: new Text('$i'), callback: (a, b) { itemsPainted.add(i); });
```

Comment:
why did this silence the error? we should put the types in, or use _ and __ to use the line in  analyzer that silences this